env.sh

```diff
@@ -1,23 +1,23 @@
 export INTERACTIVE=false
 
 # The new version being defined, i.e.: 1.4.0
-export NEW_VERSION="1.4.0-rc3"
+export NEW_VERSION="1.4.1"
 
 # The current version to be used as a source for the new one, i.e.: 1.4.0-rc2
-export CUR_VERSION="1.4.0-rc2"
+export CUR_VERSION="1.4.0-rc3"
 
 # The latest released version to be replaced (non rc), i.e.: 1.3.0
 export REPLACES_VERSION="1.3.0"
 
 # The tag to be used for the skupper-router image
-export SKUPPER_ROUTER_TAG="2.4.0"
+export SKUPPER_ROUTER_TAG="2.4.1"
 
 # The tag to be used for the control plane images:
 # site-controller, service-controller, config-sync, flow-collector
-export SKUPPER_CONTROL_TAG="1.4.0-rc3"
+export SKUPPER_CONTROL_TAG="1.4.1"
 
 # Comma separated list of versions to be skipped (optional)
 # example:
 # SKIP_VERSIONS="1.4.0-rc2,1.4.0-rc3"
 #unset SKIP_VERSIONS
-export SKIP_VERSIONS="1.4.0-rc2"
+export SKIP_VERSIONS="1.4.0-rc2,1.4.0-rc3"
```

Comment:
is it necessary to add 1.4.1-rc4 here as well?

Reply:
there is no 1.4.0-rc4 version in the operator, that's why

Reply:
No, because the operator was not released for rc1.